src/xo-app/vm/tab-disks.js

```diff
@@ -516,11 +660,52 @@ export default class TabDisks extends Component {
       isAdmin || (resourceSet == null && isVmAdmin)
   )
 
+  individualActions = [
+    {
+      handler: this._migrateVdi,
+      icon: 'vdi-migrate',
+      label: _('vdiMigrate'),
+    },
+    {
+      disabled: (vdi, userData) => {
+        const vbd = find(
+          userData.vbds,
+          vbd => vbd.VDI === vdi.id && vbd.VM === userData.vm.id
+        )
+        if (vbd !== undefined) {
+          return vbd.attached
+        }
+        return true
+      },
+      handler: deleteVbd,
+      icon: 'vdi-forget',
+      label: _('vdiForget'),
+      level: 'danger',
+    },
+    {
+      disabled: (vdi, userData) => {
+        const vbd = find(
+          userData.vbds,
+          vbd => vbd.VDI === vdi.id && vbd.VM === userData.vm.id
```

Comment:
The VBD's VM should already be this VM thanks to [this](https://github.com/vatesfr/xo-web/blob/master/src/xo-app/vm/index.js#L56-L58), right?